src/App.js

```diff
@@ -29,14 +31,35 @@ console.log = function getMessages(message) {
   }
 };
 
-export default function App() {
-  const classes = useStyles();
+export async function decodeFSH(encodedFSH) {
+  if (encodedFSH.text === undefined) {
+    return 'Edit FSH here!';
+  } else {
+    const promisedURL = await expandLink(encodedFSH);
+    const sliced64 = promisedURL.long_url.slice(38);
+    const displayText = inflateSync(Buffer.from(sliced64, 'base64')).toString('utf-8');
+    return displayText;
+  }
+}
 
+export default function App(props) {
+  const classes = useStyles();
+  const text64 = props.match.params;
+  let decodedText = null;
   const [doRunSUSHI, setDoRunSUSHI] = useState(false);
-  const [inputText, setInputText] = useState('Edit FSH here!');
+  const [inputText, setInputText] = useState('Edit FSH Here!');
   const [outputText, setOutputText] = useState('Your JSON Output Will Display Here: ');
   const [isOutputObject, setIsOutputObject] = useState(false);
 
+  useEffect(() => {
+    async function waitForFSH() {
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do you know what we need to disable this lint rule? Do you know if there's a way to change how we write this function to avoid disabling the lint rule?

I did look at this a bit and it looks possible, but maybe I'm missing an edge case, so we can talk about this one.